source/mesh_deformation/interface.cc

```diff
@@ -1,5 +1,5 @@
 /*
-  Copyright (C) 2020 by the authors of the ASPECT code.
+  Copyright (C) 2014 - 2020 by the authors of the ASPECT code.
```

Comment:
Sorry for removing this! Originally it said `2011` instead of `2014`, is that important?

Reply:
No big deal, I think the 2011 was wrong as well, this file was added in 2014 (I checked the git log). Ready from my side.